src/Console/Commands/CrudModelBackpackCommand.php

```diff
@@ -51,37 +51,63 @@ class CrudModelBackpackCommand extends GeneratorCommand
      */
     public function handle()
     {
-        $name = $this->qualifyClass($this->getNameInput());
+        $name = $this->getNameInput();
+        $namespaceApp = $this->qualifyClass($this->getNameInput());
+        $namespaceModels = $this->qualifyClass('/Models/'.$this->getNameInput());
 
+        // Check if exists on app or models
+        $existsOnApp = $this->alreadyExists($namespaceApp);
+        $existsOnModels = $this->alreadyExists($namespaceModels);
+
+        // If no model was found, we will generate the path to the location where this class file
+        // should be written. Then, we will build the class and make the proper replacements on
+        // the stub files so that it gets the correctly formatted namespace and class name.
+        if (! $existsOnApp && ! $existsOnModels) {
+            $this->makeDirectory($namespaceModels);
+
+            $this->files->put($this->getPath($namespaceModels), $this->sortImports($this->buildClass($namespaceModels)));
+
+            $this->info($this->type.' created successfully.');
+
+            return;
+        }
+
+        // If it was found on both namespaces, we'll ask user to pick one of them
+        if ($existsOnApp && $existsOnModels) {
+            $result = $this->choice('Multiple models with this name were found, which one do you want to use?', [
+                1 => "Use $namespaceApp",
+                2 => "Use $namespaceModels",
+            ]);
+
+            // Disable the namespace not selected
+            $existsOnApp = $result === 1;
+            $existsOnModels = $result === 2;
+        }
+
+        $name = $existsOnApp ? $namespaceApp : $namespaceModels;
         $path = $this->getPath($name);
 
-        // First we will check to see if the class already exists. If it does, we don't want
-        // to create the class and overwrite the user's code. We just make sure it uses CrudTrait
-        // We add that one line. Otherwise, we will continue generating this class' files.
-        if ((! $this->hasOption('force') ||
-             ! $this->option('force')) &&
-             $this->alreadyExists($this->getNameInput())) {
+        // As the class already exists, we don't want to create the class and overwrite the
+        // user's code. We just make sure it uses CrudTrait. We add that one line.
+        if (! $this->hasOption('force') || ! $this->option('force')) {
             $file = $this->files->get($path);
-            $file_array = preg_split('/(\r\n)|\r|\n/', $file);
+            $lines = preg_split('/(\r\n)|\r|\n/', $file);
 
             // check if it already uses CrudTrait
             // if it does, do nothing
-            if (Str::contains($file, [$this->crudTrait])) {
-                $this->info('Model already exists and uses CrudTrait.');
+            if (Str::contains($file, $this->crudTrait)) {
+                $this->error('Model already uses CrudTrait.');
```

Comment:
I wouldn't call this an `error`. It still does what it's supposed to do, even if that's nothing 😀
```suggestion
                $this->info('Model already used CrudTrait.');
```